Octokit/Models/Request/Enterprise/UpdatePreReceiveEnvironment.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System.Diagnostics;
+
+namespace Octokit
+{
+    /// <summary>
+    /// Describes an update to an existing pre-receive environment.
+    /// </summary>
+    [DebuggerDisplay("{DebuggerDisplay,nq}")]
+    public class UpdatePreReceiveEnvironment : NewPreReceiveEnvironment
```

Comment:
I think for consistency with the rest of the code base, it would be good to avoid an `Update` request inheriting from the `New` request, and instead implement the required fields directly in this class

Reply:
I split these up, also updated the tests to account for optional values